commands/user.go

```diff
@@ -174,6 +174,24 @@ var VerifyUserEmailWithoutTokenCmd = &cobra.Command{
 	Args:    cobra.MinimumNArgs(1),
 }
 
+var PromoteGuestToUserCmd = &cobra.Command{
+	Use:     "promote [guests]",
+	Short:   "Promote guests to users",
+	Long:    "Convert a guest into a regular user.",
+	Example: "  user promote guest1",
```

Comment:
can we use the new example format to show that this can be used with one or multiple users?